app/src/main/kotlin/com/wire/android/feature/auth/registration/CreateAccountActivity.kt

```diff
@@ -10,11 +10,22 @@ class CreateAccountActivity : AppCompatActivity(R.layout.activity_create_account
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        replaceFragment(R.id.createAccountLayoutContainer, CreateAccountFragment.newInstance(), false)
+        if (savedInstanceState == null) {
```

Comment:
Is this part of the official style? Because I feel like this is too verbose (java style). 
Something like this is what I see all the time over there, plus it is more readable and concise:

```kotlin
        if (savedInstanceState == null) {
            replaceFragment(R.id.createAccountLayoutContainer, CreateAccountFragment.newInstance(), false)
        }
        initBackButton()
```

Another alternative:
```kotlin
savedInstanceState ?: replaceFragment(R.id.createAccountLayoutContainer, CreateAccountFragment.newInstance(),false)

initBackButton()
```

Can we discuss all these style guidelines?